client/state/data-layer/wpcom/read/tags/index.js

```diff
@@ -24,20 +22,22 @@ export function requestTags( store, action, next ) {
 		? `/read/tags/${ action.payload.slug }`
 		: '/read/tags';
 
-	store.dispatch( http( {
-		path,
-		method: 'GET',
-		apiVersion: '1.2',
-		onSuccess: action,
-		onFailure: action,
-	} ) );
+	store.dispatch(
+		http( {
+			path,
+			method: 'GET',
+			apiVersion: '1.2',
+			onSuccess: action,
+			onFailure: action,
+		} )
+	);
 
 	next( action );
 }
 
 export function receiveTagsSuccess( store, action, next, apiResponse ) {
 	let tags = fromApi( apiResponse );
-	if ( ! apiResponse || ! apiResponse.tag && ! apiResponse.tags ) {
+	if ( ! apiResponse || ( ! apiResponse.tag && ! apiResponse.tags ) ) {
```

Comment:
```js
1 === 2 ? 3 : 4;
```

does this evaluate to `false` or `4`? can you answer with confidence without looking it up or trying it out?

exactly why I love this about the "extra" parentheses. :smile:

(it evaluates to `4`)

```js
1 === 2 ? 3 : 4 // 4
1 === (2 ? 3 : 4) // false
```